gym_cap/envs/cap_env.py

```diff
@@ -524,7 +525,13 @@ def render(self, mode='human'):
                         [10+SCREEN_W//2, 10+SCREEN_H//2], [SCREEN_W//2-10, SCREEN_H//2-10])
         self._env_render(self._env,
                         [10, 10+SCREEN_H//2], [SCREEN_W//2-10, SCREEN_H//2-10])
-
+        self._env_render(self.blue_memory,
```

Comment:
Render memory when the memory_mode is either fog or full

Reply:
i updated cap_env (issue 38)